<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>System configuration</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="1" column="2">
         <widget class="QPushButton" name="pbStartConfiguration">
          <property name="text">
           <string> Start configuration  </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>System test configuration</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Test mode</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cbTestMode">
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>DATA FORM ADC STREAM</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>DDR3 FPGA AS MEMORY</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>DDR3 FPGA AS FIFO</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>PCIE DATA RATE TEST</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>Board and FPGA configuration</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0" colspan="2">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QCheckBox" name="cbSlot6">
                 <property name="text">
                  <string>SLOT6</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QCheckBox" name="cbSlot3">
                 <property name="text">
                  <string>SLOT3</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QCheckBox" name="cbSlot2">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>SLOT2</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QCheckBox" name="cbSlot4">
                 <property name="text">
                  <string>SLOT4</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QCheckBox" name="cbSlot5">
                 <property name="text">
                  <string>SLOT5</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="0" column="5">
                <widget class="QCheckBox" name="cbSlot1">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>SLOT1</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0" colspan="2">
              <layout class="QGridLayout" name="gridLayout_16">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Board mask</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="leBoardMask">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="font">
                  <font>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>0x3D</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>FPGA ADC mask</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="leFpgaMask">
                 <property name="text">
                  <string>0x3</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>ADC channels mask</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="leAdcMask">
                 <property name="text">
                  <string>0xF</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>ADC channels start</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="cbAdcStartSource">
                 <item>
                  <property name="text">
                   <string>0x3 - Software</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0x2 - Hardware</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>AC_SYNC configuration</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>FD, MHz</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>SEL_CLK_OUT</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Working mode</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="cbSyncSelClkOut">
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>0x0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0x1</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="cbSyncMode">
                 <property name="currentIndex">
                  <number>2</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>0x0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0x1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0x2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0x3</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QPushButton" name="pbSyncConfigure">
                 <property name="text">
                  <string>Configure AC_SYNC</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>FO, MHz</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="cbSyncFO">
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>10.0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>56.0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>120.0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>400.0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>448.0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>480.0</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="cbSyncFD">
                 <property name="currentIndex">
                  <number>1</number>
                 </property>
                 <item>
                  <property name="text">
                   <string>400.0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>448.0</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>488.72</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>480</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>DMA configuration</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="2" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>DMA block size</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>DMA buffers count</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>DMA blocks count</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>DMA channel</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="leDmaChannel">
                 <property name="text">
                  <string>0x0</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="leDmaBlocksCount">
                 <property name="text">
                  <string>64</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="leDmaBlockSize">
                 <property name="text">
                  <string>0x100000</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="leDmaBuffersCount">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="ptTrace">
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>ADC data test</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>ADC TEST TRACE BUFFER</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="ptTraceAdc">
             <property name="textInteractionFlags">
              <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Update period, ms</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leUpdateInfoPeriodAdc">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>1000</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbStartAdcTest">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbStopAdcTest">
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>PCI Express data rate</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="2" column="3">
         <widget class="QLineEdit" name="leUpdateInfoPeriod">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>1000</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pbStartPcieTest">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="pbStopPcieTest">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Update period, ms</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="6">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>PCIE DATA TRANSFER ERROR COUNTERS</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableError"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>PCIE DATA TRANSFER RATE, MB/s</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableRate"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
